runtime/Cargo.toml

```diff
@@ -57,6 +58,7 @@ solana-stake-program = { path = "../programs/stake", version = "=1.15.0" }
 solana-vote-program = { path = "../programs/vote", version = "=1.15.0" }
 solana-zk-token-proof-program = { path = "../programs/zk-token-proof", version = "=1.15.0" }
 solana-zk-token-sdk = { path = "../zk-token-sdk", version = "=1.15.0" }
+spl-token-2022 = { version = "=0.5.0", features = ["no-entrypoint"] }
```

Comment:
Moved to `[dev-dependencies]`, hopefully thats ok

Reply:
No, sorry. You're going to have to use the inline tools for the spl-token program id, and for Mint and TokenAccount states, you can construct slices manually inside your test instead of using Pack. (Sort of like the inline-tools native mint: https://github.com/solana-labs/solana/blob/bec97d3c6b99f686cd6d181b027eb993e9e784ad/runtime/src/inline_spl_token.rs#L86)